transactions/transactions.go

```diff
@@ -12,6 +13,74 @@ import (
 
 const maxGasLimit = 9999
 
+type SignedTransaction struct {
+	flow.Transaction
+}
```

Comment:
I felt that it doesn't make sense to replicate the fields but re-use existing type instead.